custom_components/miele/sensor.py

```diff
@@ -174,14 +174,29 @@ class MieleSensorDefinition:
     MieleSensorDefinition(
         types=[
             WASHING_MACHINE,
+            WASHER_DRYER,
+        ],
+        description=MieleSensorDescription(
+            key="targetTemperature",
+            data_tag="state|targetTemperature|0|value_raw",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            name="Target temperature",
+            native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_category=EntityCategory.DIAGNOSTIC,
+            decimals=0,
```

Comment:
```suggestion

```
Decimals is a leftover from cut&paste from another integration. Has no function in this integration. Should be removed.